src/cpp/rtps/builtin/discovery/endpoint/EDPSimple.cpp

```diff
@@ -48,16 +48,19 @@ const Duration_t edp_nack_response_delay{0, 400*1000*1000}; // ~93 milliseconds
 const Duration_t edp_nack_supression_duration{0, 50*1000*1000}; // ~11 milliseconds
 const Duration_t edp_heartbeat_response_delay{0, 50*1000*1000}; // ~11 milliseconds
 
+const int32_t edp_initial_reserved_caches = 100;
```

Comment:
This may be too high a value for the general case. As this PR sets an infinite maximum value, I think this can be reduced. I don't consider it a big issue, though, as in the near future we are going to give the user full control over the history configuration of the builtin endpoints.

Reply:
Done